src/predicate.rs

```diff
@@ -82,20 +114,104 @@ impl<'a> Predicate for Attr<&'a str, &'a str> {
     }
 }
 
-impl<'a> Predicate for Attr<&'a str, ()> {
```

Comment:
I agree that this impl needs to go, but could you restore this for now (add a "deprecated" comment maybe) so I don't have to bump the minor version number right now? I'll remove the impl with other breaking changes.

Reply:
Yes I will do this